Formula/r/rkdeveloptool.rb

```diff
@@ -0,0 +1,25 @@
+class Rkdeveloptool < Formula
+  desc "Tool for flashing and debugging Rockchip devices over USB"
+  homepage "https://github.com/rockchip-linux/rkdeveloptool"
+  url "https://github.com/rockchip-linux/rkdeveloptool/archive/304f073752fd25c854e1bcf05d8e7f925b1f4e14.tar.gz"
+  version "1.32"
+  sha256 "389ba41af6986c16f1eeebdc1febcb0bf4b8acb7abd694d3d652e78504215843"
+  license "GPL-2.0-only"
+  head "https://github.com/rockchip-linux/rkdeveloptool.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkgconf" => :build
+  depends_on "libusb"
+
+  def install
+    system "autoreconf", "-i"
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    bin.install "rkdeveloptool"
+  end
+
+  test do
+    assert_equal "rkdeveloptool ver 1.32", shell_output("#{bin}/rkdeveloptool -v").strip
```

Comment:
```suggestion
    assert_equal version.to_s, shell_output("#{bin}/rkdeveloptool -v")
```
We need a better test that uses basic functionality of an app
> We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

See more details here: https://github.com/rockchip-linux/rkdeveloptool

Reply:
> We need a better test that uses basic functionality of an app

This software does not have the possibility to test the functionality, as an external device must be connected and set to a special mode. I don't think this is a good idea as it is not obvious when installing the software

Reply:
In this case please add a comment that explains why the test just asserts version output (something like `# This program requires an external device for proper testing`)

Reply:
> This software does not have the possibility to test the functionality, as an external device must be connected and set to a special mode. 

In that case it's fine to check for an expected error.
> I don't think this is a good idea as it is not obvious when installing the software

Tests only run in CI or when people run `brew test <formula>`

Reply:
> In that case it's fine to check for an expected error.

I can do as you ask.
It would be good if you can explain for me the difference between the version check output and the program output: `not found any devices!`
and can I do this as an additional test?

Reply:
@IgorKha You can ignore comments from @botantony here as they are not a Homebrew maintainer. Do listen to @SMillerDev though.